Gemfile

```diff
@@ -71,9 +71,7 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
-#gem 'blacklight', '~> 6.12.0'
-
-gem 'blacklight', '>= 7.0'
+gem 'blacklight', '~> 7.10'
```

Comment:
This appears related to the source of the problem with uglifier. There was an incidental upgrade from Blacklight 7.1 to 7.10 in f94ffa53e. There were upstream changes to blacklight.js that introduce ES6 arrow functions.

https://github.com/projectblacklight/blacklight/commit/c10b951522f621b01c8892d8b2a503fea56a9b9d#diff-5bbd7742449f1a6db615586a27502949
https://github.com/projectblacklight/blacklight/commit/ce9a084f959eebb579b93e2ff6214b82b4767766#diff-5bbd7742449f1a6db615586a27502949

That's potentially a bug in upstream (breaking 5.x by using uglifier without es6 enabled) or potentially just in our application.rb, based on when the `rails new` was run (and the generators therein).

These are different than the sassc + docker / CPU architecture bug:

 * https://northsail.io/articles/ruby-sassc-illegal-instruction
 * https://github.com/sass/sassc-ruby/issues/146

Upgrading ffi and sassc solve the latter, while enabling harmony/es6 solves the former. No telling what else changed in Blacklight between 7.1 and 7.10, though.